pkg/urbit/vere/io/fore.c

```diff
@@ -69,6 +69,32 @@ _fore_inject(u3_auto* car_u, c3_c* pax_c)
   u3z(ovo);
 }
 
+/* _fore_import(): form an ovum from jammed archive at [pax_c] and inject it.
+*/
+static void
+_fore_import(u3_auto* car_u, c3_c* pax_c)
+{
+  // With apologies
+  u3_noun arc = u3ke_cue(u3m_file(pax_c));
+  c3_c * b64_c = u3r_string(u3do("crip", u3do("en-base64:mimes:html", arc)));
```

Comment:
This transfers `arc` and the product of `+en-base64`, but leaks the product of `+crip` (`u3r_string()` has retain semantics). You'll need to assign that to a variable and `u3z()` it once you're done.

Reply:
You'll also need to call `c3_free()` on `b64_c`